core/sql/executor/ExStats.cpp

```diff
@@ -3678,9 +3292,8 @@ void ExPartitionAccessStats::getVariableStatsInfo(char * dataBuffer,
 	      "AnsiName: %s PhysName: %s BuffersSize: %u BuffersSent: %u BuffersRcvd: %u NumMessages: %Ld MsgBytes: %Ld StatsBytes: %Ld MsgBytesSent: %Ld MsgBytesRcvd: %Ld ",
 	      ansiName_, fileName_,
 	      bufferStats()->sendBufferSize(), bufferStats()->sentBuffers().entryCnt(), bufferStats()->recdBuffers().entryCnt(), 
-	      fsDp2MsgsStats()->getNumMessages(),
-	      fsDp2MsgsStats()->getMessageBytes(),
-	      fsDp2MsgsStats()->getStatsBytes(),
+	      exeSEStats()->getNumIOCalls(),
+	      exeSEStats()->getNumIOBytes(),
```

Comment:
Thanks Selva. Noticed that we have 10 format specifiers (including StatsBytes) and only 9 arguments hence the question. Line 4522 has similar logic ( though it's sprintf). Maybe I'm missing something here - will check on the code again as well as str_sprintf.